packages/account/src/Containers/Account/account.tsx

```diff
@@ -32,12 +33,12 @@ const Account = observer(({ history, location, routes }: TAccountProps) => {
         should_allow_poinc_authentication,
         is_passkey_supported,
     } = client;
-    const { toggleAccountSettings, is_account_settings_visible, is_mobile, is_desktop } = ui;
-
+    const { toggleAccountSettings, is_account_settings_visible } = ui;
+    const { isMobile } = useDevice();
     // subroutes of a route is structured as an array of arrays
     const subroutes = flatten(routes.map(i => i.subroutes));
     const selected_content = subroutes.find(r => matchRoute(r, location.pathname));
-    const should_remove_passkeys_route = is_desktop || (is_mobile && !is_passkey_supported);
+    const should_remove_passkeys_route = !isMobile || (isMobile && !is_passkey_supported);
```

Comment:
What is the requirement for tablet view? Coz `!isMobile` means Tablet view as well

Reply:
we will hide passkeys for tablet view as well